src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesSlave.java

```diff
@@ -538,6 +543,51 @@ public static Builder builder() {
         return new Builder();
     }
 
+    public void annotateTtl(TaskListener listener) {
+        var kubernetesCloud = getKubernetesCloud();
+        Optional.ofNullable(kubernetesCloud.getGarbageCollection()).ifPresent(gc -> {
+            var ns = getNamespace();
+            var name = getPodName();
+            var l = Instant.now();
+            if (lastRefresh == 0
+                    || Duration.between(Instant.ofEpochMilli(lastRefresh), Instant.now())
+                                    .compareTo(gc.getDurationTimeout().dividedBy(2))
+                            > 0) {
+                try {
+                    kubernetesCloud
+                            .connect()
+                            .pods()
+                            .inNamespace(ns)
+                            .withName(name)
+                            .edit(p -> {
```

Comment:
IIRC this call is vulnerable to race conditions (ignore modification serial number), and you should explicitly patch if you can.